GoogleForCreators/web-stories-wp
# Dashboard: Fix Detail View Hard Refresh Bug; Improve overall loading performance

## Summary

Fixes a bug where refreshing the browser on the template detail view prevents the page from reappearing.

Improves overall loading performance of the template detail view loading the metadata first, then the page cards.

## Relevant Technical Choices

Ensures the templates are pre-loaded. Uses a loading flag to prevent multiple requests.

## User-facing changes

Hard refreshing the browser on the template detail view should work again.

## Testing Instructions

1. Visit Explore Templates from Dashboard
2. Click View on any Template
3. Refresh the Browser and see the page reloads correctly

---



#4246
